### ISSUE TITLE
Missing some build instructions on README?

### ISSUE THREAD

**Issue by shqld** [fastly/js-compute-runtime#78]
Missing some instructions on README for the build?

I encountered errors during the following build commands on [README.md](https://github.com/fastly/js-compute-runtime/blob/dbd9/README.md#working-with-the-js-compute-runtime-source):

```sh
$ git submodule update --recursive --init
$ (cd c-dependencies/spidermonkey && sh download-engine.sh)
$ cargo build --release // ERROR
```

Not sure if this is common for other people, but I share what I've done, for the record.

- Install `cbindgen`
  https://github.com/eqrion/cbindgen#quick-start
- Install `wasi-sdk` and export the sdk path
  https://github.com/WebAssembly/wasi-sdk#install
  `$ export WASI_CXX=${WASI_SDK_PATH}/bin/clang++`
- Modify Makefile
  ```diff
  -ifdef DEBUG
  +DEBUG ?= false
  +ifneq ($(DEBUG), false)
  ```
  *cargo exports `DEBUG=false` when running `build.rs` with `--release` flag

Environment:
  - hash: `dbd9386`
  - platform: M1 Mac (darwin, arm64)

**Comment by black2d** [comment]
I had similar issues I submitted a pr #70 to update the readme a while back but it hasn't merged yet.

**Comment by elliottt** [comment]
I think everything here was addressed by merging #70 except for the comments about `DEBUG`. I tried building locally without the Makefile changes both debug and release builds and didn't run into any problems, do you have steps I could follow to reproduce them?

**Comment by elliottt** [comment]
I was able to reproduce the problem with `DEBUG`, and will fix it in #98.

**Comment by jameysharp** [comment]
Now that #101 is merged (instead of #98), and in conjunction with the README improvements in #70, I believe this issue can be closed. Thanks for your report and please do let us know if you have further issues!
